_posts/2021-09-02-gsoc-final-report-estefania.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post
+title: "GSoC Report: Curvature for MDA Universes"
+---
+
+The main goal of my [GSoC project](https://summerofcode.withgoogle.com/projects/#5098282306502656)
+was to develop a tool to calculate membrane curvature
+from MD simulations using [MDAnalysis]. We
+aimed to have the following features in the membrane curvature analysis tool:
+
+- Surfaces derived from an `AtomGroup` of reference.
+- Calculation of mean and Gaussian curvature.
+- Implement multiframe and averaged-over-frames analysis.
+- Plug-and-play with visualization libraries to obtain 2D curvature profiles.
+- Data visualization made easy.
+
+<img
+src="https://ojeda-e.github.io/assets/images/patches.png"
+alt="membrane_patches" />
+
+
+# Why Membrane Curvature?
+
+In the wide range of tools that are available to analyze Molecular Dynamics (MD)
+simulations, user-friendly, actively-maintained, and well-documented tools to
+calculate membrane curvature are still difficult to find. I was motivated to
+share a tool to calculate membrane curvature that I initially developed as part
+of my PhD at the [Biocomputing Group](https://ucalgary.ca/biocomputing/home).
+Membrane curvature is a phenomenon that can be investigated via MD simulations, and
+I had an interest to share this tool with the wider MD community. 
+
+
+# Contributions
+Keeping in mind the goals and motivations behind my GSoC project,
+I would like to hightlight three areas of contributions that were key to the
+MembraneCurvature MDAnalysis tool: [Core functions](#core-functions-34-40-44),
+[AnalysisBase](#analysis-base-43-48), and [Documentation](#documentation-57-62-64-69).
+
+## Core functions ([#34], [#40], [#44])
```

Comment:
Did you test that `[#34]` resolves as a link? Maybe `[#34][]` does the job?

Reply:
Hmmm... I tested all the PR links, indeed. It is slow but links accordingly.